exercises/ruby-1-a/ruby_1_a_test.rb

```diff
@@ -1,33 +1,53 @@
+# frozen_string_literal: true
+
 require 'minitest/autorun'
 require_relative 'ruby_1_a'
 
-# Common test data version: 1.2.0 4fc1acb
-class SentenceTest < Minitest::Test
-  def test_one_word_with_one_vowel
-    assert_equal "a", word_with_most_vowels("a")
+class ShoppintCartTest < Minitest::Test
+  def test_initial_total_amount
+    assert_equal 0, ShoppingCart.new.total_amount
   end
 
-  def test_one_word_with_one_vowel_and_one_consonant
-    assert_equal "at", word_with_most_vowels("at")
+  def test_one_item
+    cart = ShoppingCart.new
+    cart.add(:STAPOT)
+    assert_equal 10, cart.total_amount
   end
 
-  def test_one_word_with_two_vowels_and_one_consonant
-    assert_equal "tea", word_with_most_vowels("tea")
+  def test_two_items
+    cart = ShoppingCart.new
+    cart.add(:STAPOT)
+    cart.add(:STARIC)
+    assert_equal 40, cart.total_amount
   end
 
-  def test_longest_word_is_not_word_with_most_vowels
-    assert_equal "area", word_with_most_vowels("the area of a circle")
+  def test_list_items
+    cart = ShoppingCart.new
+    cart.add(:STARIC)
+    cart.add(:STAPOT)
+    assert_equal 'Potatoes, Rice', cart.items_list
   end
 
-  def test_last_word_is_word_with_most_vowels
-    assert_equal "vegetables", word_with_most_vowels("extremely fresh vegetables")
+  def test_items_ordered_alphabetically
+    cart = ShoppingCart.new
+    cart.add(:STAPOT)
```

Comment:
I think these need to be swapped round, so that the order added to the basket is **not** alphabetical. So Rice should be added before Potatoes. (And we should switch it in the previous test also)